src/client/Client.cc

```diff
@@ -3090,6 +3091,32 @@ void Client::put_inode(Inode *in, int n)
   }
 }
 
+void Client::delay_put_inodes(bool wakeup)
+{
+  ceph_assert(ceph_mutex_is_locked_by_me(client_lock));
+
+  std::map<Inode*,int> release;
+  {
+    std::scoped_lock dl(delay_i_lock);
+    release.swap(delay_i_release);
+    delay_i_release.clear();
```

Comment:
nit, `delay_i_release` should be empty. so no need to clear it.

Reply:
Removed it. Thanks.